data.sql

```diff
@@ -3,4 +3,13 @@
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03', 0, true, 10.23);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, true, 8.00);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, true, 11.00);
\ No newline at end of file
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, true, 11.00);
+
+/* Update table's data. */
+
```

Comment:
- [ ] Good job so far 👏 Please note that you have forgotten to record data of one animal which is `Charmander` as it is in the project requirements. 
![image](https://user-images.githubusercontent.com/90222110/228605687-5ec8c61a-6738-4f1b-b9b3-feaf02ef02c0.png)

Therefore, I would suggest that you record it there too 😃 However I saw that in your screenshots it was there among the results, I don't if you forgot to put it here only but if it is the case, you just need to record it here too 👍

Reply:
@BertrandConxy, 
So sorry, I forgot to record it in the file. I will do this now.
Thanks